src/main/java/database/managers/PlayerCreatorManager.java

```diff
@@ -0,0 +1,110 @@
+package database.managers;
+
+import interfaces.State;
+import playercreation.PlayerCreatorInteractor;
+import playercreation.PlayerQuestion;
+import playercreation.PlayerQuestionState;
+import playercreation.PlayerConfirmState;
+import core.StateManager;
+
+import java.util.ArrayList;
+
+public class PlayerCreatorManager extends StateManager {
+    /** A controller class that helps create a new Player character at the beginning of the game.
+     * Attributes:
+     * currPlayerQuestion: The current question about the Player character being asked of the user. An enum of
+     *                     type PlayerQuestion.
+     * completedQuestions: A List of completed PlayerQuestions that have already been filled by the user. Does not
+     *                     include CONFIRM or COMPLETE.
+     * creatorInteractor: A PlayerCreatorInteractor that stores information about the Player until all questions have
+     *                    been answered by the user.
+     */
+
+    private PlayerQuestion currPlayerQuestion;
+    private final ArrayList<PlayerQuestion> completedQuestions;
+    private final PlayerCreatorInteractor creatorInteractor;
```

Comment:
recommend to put this in a factory to create the states instead of creating them in here so you avoid introducing this dependency and also all the state dependencies